DirectX11/IniHandler.cpp

```diff
@@ -1218,6 +1221,63 @@ static void EnumerateCustomShaderSections(IniSections &sections, wchar_t *iniFil
 		customShaders[shader_id];
 	}
 }
+
+static void ParseSamplerState(CustomShader *shader, const wchar_t *section, wchar_t *iniFile)
+{
+	D3D11_SAMPLER_DESC* desc = &shader->sampler_desc;
+	wchar_t setting[MAX_PATH];
+	wchar_t key[32];
+	int i;
+	bool found;
+
+	memset(desc, 0, sizeof(D3D11_SAMPLER_DESC));
+
+	//TODO: do not really understand the difference between normal and comparison filter 
```

Comment:
After few test with different configs i find the current configuration produces the best quality. Tried witcher with 4k interlacing (game res full HD vs 2K). Both looks very good, and almost the same (fullhd) is a bit blurry, but maybe i can add a deblurr filter or somerhing like that. However this is a very small drawback for constant 60 fps :-)